### DISCUSSION TITLE
e2e tests

### DISCUSSION THREAD

**Discussion by xdxxxdx** [eclipse-edc/FederatedCatalog#185] [Q&A]
As the docker compose file is deleted by the pull request:
https://github.com/eclipse-edc/FederatedCatalog/commit/e95bdbca1493135dd986d3e5190b0df5694e8195#diff-5a009985c3c2b86314420cbabdc342063e0b66628bf25b9f879911085079a2c7
I am wondering how the https://github.com/eclipse-edc/FederatedCatalog/tree/main/system-tests/end2end-test#test-execution
is supposed to run.

Thanks so much

**Comment by paullatzelsperger** [comment]
we have a way of spinning up runtimes from within the JUnit runner using dedicated `URLClassLoaders`. The documentation is wrong though, and it should be removed.
